### ISSUE TITLE
Remote URL playlist to a list (m3u/pls) doesn't work with local files

### ISSUE THREAD

**Issue by janek** [AzuraCast/AzuraCast#5703]
### Installation Method

Docker Installation

### AzuraCast Release Channel

Stable Channel

### Current AzuraCast Version

0.17.2

### What happened?

I'm using the Remote URL with the "_Playlist (M3U/PLS) URL_"  option. I want to point to a file with a list of songs to play. I can get it to work with a list of mp3 URLs hosted elsewhere, but not with a list of local filepaths (URIs).

✅ **This works:**
```
https://www.soundhelix.com/examples/mp3/SoundHelix-Song-1.mp3
https://www.soundhelix.com/examples/mp3/SoundHelix-Song-2.mp3
https://www.soundhelix.com/examples/mp3/SoundHelix-Song-3.mp3
```

❌  **This doesn't work:**
```
1.mp3
2.mp3
3.mp3
```

**Liquidsoap log output for the above input playlist:**
```
2022/08/31 10:47:39 [procol.external:3] No known file extension for mime: text/plain; charset=utf-8
2022/08/31 10:47:39 [request:3] Nonexistent file or ill-formed URI "1.mp3"!
2022/08/31 10:47:39 [procol.external:3] No known file extension for mime: text/plain; charset=utf-8
2022/08/31 10:47:39 [request:3] Nonexistent file or ill-formed URI "2.mp3"!
2022/08/31 10:47:40 [procol.external:3] No known file extension for mime: text/plain; charset=utf-8
2022/08/31 10:47:40 [request:3] Nonexistent file or ill-formed URI "3.mp3"!
2022/08/31 10:47:40 [procol.external:3] No known file extension for mime: text/plain; charset=utf-8
```

**Sanity checks: the files are there:**
<img width="1102" alt="image" src="https://user-images.githubusercontent.com/2146927/187661927-3e0a5eec-0ab3-493f-98ab-c6e4c78225c2.png">

**Why I was assuming this would work** 
1. When I make a "normal" ("Song-Based") playlists that includes the example tracks in question, and then export with the "Export as M3U" button, I get exactly the thing I'm trying to get to work:
```
1.mp3
2.mp3
3.mp3
```
It works fine when it's defined locally, but if I provide exactly the same definition via a remote file, AzuraCast doesn't see them anymore. 

2. `Nonexistent file or ill-formed URI "3.mp3"` suggest that you can operate on files

**What I tried:**
- I looked through the docs a few times, but I think at the moment this is still underdocumented
- I read the relevant code to the extent that I could and didn't find why it shouldn't work 
- asked on Discord twice but didn't get any answers
- scanned all relevant issues on GitHub

Thank you in advance, appreciate your time.

**Comment by kemecs** [comment]
Thank you for putting this into words!

I was trying to use this feature like you, but I ran into the same problem in every version of Azura.

I hope the guys will have a solution for this.

**Comment by BusterNeece** [comment]
@janek I don't think this is a problem that can be (or should be) solved by AzuraCast itself.

The easy solution here would be to have the generated remote playlist include the full path to the relevant files, i.e. `/var/azuracast/stations/(station_name)/media/1.mp3`, which would then give Liquidsoap sufficient information to know where to look. You can also take advantage of a URL prefix that we have registered with Liquidsoap and prefix the files with `media:`, i.e. `media:1.mp3` in order to instruct Liquidsoap to look in the station's media directory (or to copy from a remote filesystem, if applicable).

Basically, Liquidsoap doesn't have any context when it's running that it should look for relative paths in a different directory than the one where its configuration lives (which is `/var/azuracast/stations/(station_name)/config`), and as far as I can tell there's no way to tell it to look for files in another directory than that. The solutions above will work, though, in giving it enough information to know where to look.

**Comment by janek** [comment]
Thank you for writing @BusterNeece!

Just to answer quickly before I procrastinate: I'm quite sure I tried full paths and the problem was still there. I now moved on with a workaround of using S3 to host the files, but I will not forget about this and will try to reproduce it with full paths. I'll write here if I manage.

**Comment by kemecs** [comment]
version full path /var/azuracast/stations/(station_name)/media/1.mp3 >
https://backend.folkradio.hu/static/audio/playlist_full_path.m3u

2022/09/12 17:16:19 [request:3] Nonexistent file or ill-formed URI "/var/azuracast/stations/folktest.folkradio.hu/media/stn200409.mp3"!
2022/09/12 17:16:20 [request:3] Nonexistent file or ill-formed URI "/var/azuracast/stations/folktest.folkradio.hu/media/smna_1_1_tanczene29.mp3"!


version media:1.mp3 >
https://backend.folkradio.hu/static/audio/playlist_full_media.m3u

2022/09/12 17:23:54 [request:3] Nonexistent file or ill-formed URI "/var/azuracast/stations/folktest.folkradio.hu/media/shippoglassus05.mp3"!
2022/09/12 17:23:55 [request:3] Nonexistent file or ill-formed URI "/var/azuracast/stations/folktest.folkradio.hu/media/sujpa1118.mp3"!


version of any >

https://backend.folkradio.hu/radio/playlist/pls
2022/09/12 17:24:50 [request:3] Nonexistent file or ill-formed URI "1474_szepsi_sacca_felsobodvamente1_10.mp3"!
2022/09/12 17:24:51 [request:3] Nonexistent file or ill-formed URI "hg_gyimes_fz2008_kerekes01_heg.mp3"!

**Comment by BusterNeece** [comment]
@kemecs Interesting. It's funny that it's actually resolving the `media:` URI, then failing when trying to actually read the media from the local filesystem. That makes me strongly suspect this is some sort of Liquidsoap issue. It may be worthwhile to raise this issue with them: https://github.com/savonet/liquidsoap/issues

**Comment by kemecs** [comment]
This version of "media:1mp3" works for me now (I messed something up)
https://backend.folkradio.hu/static/audio/playlist_full_media.m3u

Now it should just play the playlist content sequentially and not shuffle it.

Is there a solution for this?

Advanced Manual AutoDJ Scheduling Options:
"Merge playlist to play as a single track."
ticking this does not help..
